MyString.java

```diff
@@ -36,8 +51,23 @@ public static int countChar(String str, char ch) {
      * @return true is str1 is a subset of str2, false otherwise
      */
     public static boolean subsetOf(String str1, String str2) {
-         //// Replace the following statement with your code
-        return false;
+        if (str1 == "") {

```

Comment:
== doesn't work consistently with non primitive types use str.length() == 0 or str.equals("") for checking empty string